### ISSUE TITLE
[CLI] Syncing failed during training and manually syncing

### ISSUE THREAD

**Issue by bennevans** [wandb/client#2130]
**Description**
During training, I received the following error on 4 out of the 5 concurrent runs:
```
wandb: Network error (ReadTimeout), entering retry loop. See wandb/debug-internal.log for full traceback.        
Exception in thread Thread-5:                                                                                   
Traceback (most recent call last):                                                                       
  File "/home/bne215/anaconda3/lib/python3.8/threading.py", line 932, in _bootstrap_inner                
    self.run()                                                                                           
  File "/home/bne215/anaconda3/lib/python3.8/threading.py", line 870, in run                             
    self._target(*self._args, **self._kwargs)                                                            
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/file_stream.py", line 260, in _thread_body                                                                                          
    self._handle_response(                                                                               
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/file_stream.py", line 277, in _handle_response                                                                                      
    raise response                                                                                       
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/util.py", line 731, in request_with_retry                                                                                                        
    response.raise_for_status()                                                                          
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status                                                                                                     
    raise HTTPError(http_error_msg, response=self)                                                       
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: https://api.wandb.ai/files/benevans/domain_adaptation/1iu20q6o/file_stream                                                        
wandb: Network error resolved after 0:01:31.095979, resuming normal operation.                           
wandb: ERROR Error while calling W&B API: Error 1040: Too many connections (<Response [500]>)                  
wandb: Network error (ReadTimeout), entering retry loop. See wandb/debug-internal.log for full traceback.         
wandb: Network error resolved after 0:00:36.213254, resuming normal operation.   
```

The training continues, but wandb did not continue to sync after the fact, marking the run as "crashed".
I am able to manually sync the folder, resulting in more errors, but it seems to upload evenutally. 

When syncing one of the runs with `wandb sync wandb/run-20210427_234534-1mxdoulq/`, I get 
```                                                             
Retry attempt failed:                                                                                    
Traceback (most recent call last):                                                                       
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/old/retry.py", line 96, in __call__     
    result = self._call_fn(*args, **kwargs)                                                              
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 130,
 in execute                                                                                              
    six.reraise(*sys.exc_info())                                                                         
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/six.py", line 703, in reraise                 
    raise value                                                                                            File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 124,
 in execute                                         
    return self.client.execute(*args, **kwargs)                                                            File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52
, in execute                                                                                             
    result = self._get_result(document, *args, **kwargs)                                                   File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60
, in _get_result                                                                                         
    return self.transport.execute(document, *args, **kwargs)                                               File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.
py", line 39, in execute                            
    request.raise_for_status()                                                                             File "/home/bne215/anaconda3/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_st
atus                                                
    raise HTTPError(http_error_msg, response=self)                                                       
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql 
wandb: Network error (HTTPError), entering retry loop. See wandb/debug-internal.log for full traceback.
Retry attempt failed:                               
Traceback (most recent call last):                                                                       
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/old/retry.py", line 96, in __call__
    result = self._call_fn(*args, **kwargs)                                                                File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 130,
 in execute                                         
    six.reraise(*sys.exc_info())                                                                         
  File "/home/bne215/anaconda3/lib/python3.8/site-packages/six.py", line 703, in reraise
    raise value                                                                                            File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 124,
 in execute                                         
    return self.client.execute(*args, **kwargs)                                                            File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52
, in execute                                                                                             
    result = self._get_result(document, *args, **kwargs)                                                   File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60
, in _get_result                                                                                         
    return self.transport.execute(document, *args, **kwargs)                                               File "/home/bne215/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.
py", line 39, in execute                            
    request.raise_for_status()                                                                             File "/home/bne215/anaconda3/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_st
atus                                                
    raise HTTPError(http_error_msg, response=self)                                                       
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql 
wandb: Network error (HTTPError), entering retry loop. See wandb/debug-internal.log for full traceback.

```
and many errors like
```
429 response executing GraphQL.                     
429 response executing GraphQL.
{"error":"rate limit exceeded"}
```

**Wandb features**
I'm using `wandb.init()`, `wandb.log()`, `wandb/config.update()`, `wandb.watch()`, and logging `wandb.Image`s.

**How to reproduce**
Not clear how to reproduce. Will update if it occurs again

**Environment**
- OS: Ubuntu 20.04.2 LTS
- Environment: conda 
- Python Version: 3.8.5
- wandb version: 0.10.26

**Comment by ariG23498** [comment]
Hey @bennevans 
Thanks for filing a ticket with us. We will be looking into the ticket internally. If you find a reproducible code snippet later, do share it with us.

**Comment by vanpelt** [comment]
Hey everyone,

We just released 0.10.32 of our library that should address the issues around syncing offline runs.  I'm going to close this issue but please add comments with any errors you may be seeing with the latest release of the library.
